packages/phoenix-client/docs/source/index.md

```diff
@@ -229,6 +229,105 @@ df = pd.DataFrame({
 client.spans.log_span_annotations_dataframe(dataframe=df)
 ```
 
+### Evaluation Helpers
+
+The Phoenix Client provides helper functions to extract span data in formats optimized for RAG evaluation workflows. These helpers streamline the process of preparing data for evaluation with `phoenix.evals`.
+
+#### RAG Retrieval Evaluation
+
+Extract retrieved documents from retriever spans for relevance evaluation:
+
+```python
+from phoenix.client import Client
+from phoenix.client.helpers.evaluation import get_retrieved_documents
```

Comment:
just thinking about the namespaces a git, I wonder if `phoenix.client.spans.helpers` might be slightly better given that it relates to pulling spans. I can see it either way however.